eclipse-cdt-cloud/cdt-cloud-blueprint
# Add browser app and devcontainer setup

#### What it does
This PR adds a browser app for CDT.cloud blueprint and a devcontainer setup to run it from inside a container.
This container can later also be used for testing CDT.cloud blueprint without installing the OS dependencies for C/C++ development as well as for running CDT.cloud actually in a cloud infrastructure.

https://github.com/eclipse-cdt-cloud/cdt-cloud-blueprint/issues/10

#### How to test
* install the [Remote Dev extension pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) in VS Code
* open this repository in VS Code
* In the notification that should appear: confirm to open this folder in the remote container instead
* run `yarn && yarn browser && yarn browser start`
* CDT.cloud should be running on `127.0.0.1:3000` (VS Code should notify you on that)
